dietpi/dietpi-software

```diff
@@ -970,6 +970,12 @@ Available commands:
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/social/#wordpress'
 		aSOFTWARE_DEPS[$software_id]='88 89 webserver'
 		#------------------
+		software_id=206
+		aSOFTWARE_NAME[$software_id]='Homer'
+		aSOFTWARE_DESC[$software_id]='A homepage for your services'
+		aSOFTWARE_CATX[$software_id]=6
+		aSOFTWARE_DOCS[$software_id]='https://github.com/bastienwirtz/homer'
```

Comment:
This should point to our own docs.

Reply:
there wasn't a doc for homer yet, however i submitted a [PR](https://github.com/MichaIng/DietPi-Docs/pull/784) for it, so i can change if it gets added

Reply:
Many thanks for this 🙂!

Reply:
Usually, we will submit doc changes as well as DietPi release togher. Therefore, it should be safe to have it in already.